### DISCUSSION TITLE
Build Error: "error MSB4057: The target "GenerateGraphQLCode" does not exist in the project."

### DISCUSSION THREAD

**Discussion by Xeviva** [ChilliCream/hotchocolate#6051] [General]
I have a fresh VS 2022 install, and created in it a new console app (.net 6.0).

I followed the getting started instructions for strawberryShake to the letter, for creating a console app, I can see it has created the schema fine, but when I build I'm getting an error:

error MSB4057: The target "GenerateGraphQLCode" does not exist in the project.

Any ideas anyone?

**Comment by michaelstaib** [comment]
Did you see my comments on your issue?
